### PR TITLE
[TOOLS-3004] Update Actionable Agile JSON file with new fields

### PR DESCRIPTION
**Pull request by jytakano**

### CODE REVIEW

File: kpi-parent/kpi-testutils/src/main/java/objective/taskboard/followup/kpi/services/FakeIssueKpiAdapter.java [hunk 1]
```diff
@@ -16,20 +19,36 @@
     private KpiLevel level;
     private Map<String,String> customFieldsValues;
     private ZonedDateTime createdDate;
+    private boolean blocked;
+    private Optional<String> lastBlockReason;
+    private Set<CardTeam> teams;
+    private Optional<Version> release;
+    private Optional<String> tShirtSize;
     
     public FakeIssueKpiAdapter(
```
Comment (objective-solutions-user) [inline kpi-parent/kpi-testutils/src/main/java/objective/taskboard/followup/kpi/services/FakeIssueKpiAdapter.java]: MAJOR Constructor has 11 parameters, which is greater than 7 authorized. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AS00107)

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: core/src/main/java/objective/taskboard/domain/converter/IssueTeamService.java [hunk 2]
```diff
@@ -105,6 +106,13 @@ public CardTeam getDefaultTeam(Issue issue) {
         return mismatches;
     }
 
+    public Set<CardTeam> resolveTeamsVisibleToUser(Issue issue) {
+        List<Long> teamsVisibleToUser = userTeamService.getIdsOfTeamsVisibleToUser();
+        return resolveTeams(issue).stream()
+                .filter(team -> teamsVisibleToUser.contains(team.id))
+                .collect(toCollection(LinkedHashSet::new));
```
Comment (herbert-venancio) [inline core/src/main/java/objective/taskboard/domain/converter/IssueTeamService.java]: Suggestion: use `toLinkedHashSet` from `objective.taskboard.utils.StreamUtils`

File: actionable-agile/src/main/java/objective/taskboard/actionableagile/controller/ActionableAgileReportService.java [hunk 3]
```diff
@@ -43,6 +45,11 @@ ActionableAgileReport getData(String projectKey, ZoneId timezone, KpiLevel level
             .forEach(status -> report.addTransitionHeader(status));
         report.addExtraFieldHeader("Type");
         report.addExtraFieldHeader("Status");
+        report.addExtraFieldHeader("Blocked");
+        report.addExtraFieldHeader("Last Block Reason");
+        report.addExtraFieldHeader("Teams");
+        report.addExtraFieldHeader("Release");
+        report.addExtraFieldHeader("T-Shirt Size");
```
[the remark this answers is not in the document]
Comment (jytakano) [reply actionable-agile/src/main/java/objective/taskboard/actionableagile/controller/ActionableAgileReportService.java]: Issue description updated

**Pull request merged**